EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php

```diff
@@ -32,43 +19,4 @@ class EWSType_FindMailboxStatisticsByKeywordsResponseMessageType extends EWSType
      * @var EWSType_MailboxStatisticsSearchResultType
      */
     public $MailboxStatisticsSearchResult;
-
-    /**
-     * Provides a text description of the status of the response.
-     *
-     * @since Exchange 2013
```

Comment:
Yea, absolutely. I'm just saying that now the `@since` doc tags for `EWSType_ResponseMessageType` are invalid. Maybe they should just be removed? Maybe with a special note regarding the polymorphic extension that has a conditional inclusion date?